src/video.js

```diff
@@ -1,8 +1,35 @@
 import { videoAdapters } from './adaptermanager';
+import { getBidRequest, deepAccess } from './utils';
+
+const VIDEO_MEDIA_TYPE = 'video';
+const INSTREAM = 'instream';
+const OUTSTREAM = 'outstream';
 
 /**
  * Helper functions for working with video-enabled adUnits
  */
-export const videoAdUnit = adUnit => adUnit.mediaType === 'video';
+export const videoAdUnit = adUnit => adUnit.mediaType === VIDEO_MEDIA_TYPE;
 const nonVideoBidder = bid => !videoAdapters.includes(bid.bidder);
-export const hasNonVideoBidder = adUnit => adUnit.bids.filter(nonVideoBidder).length;
+export const hasNonVideoBidder = adUnit =>
+  adUnit.bids.filter(nonVideoBidder).length;
+
+/*
+ * Validate that the assets required for video context are present on the bid
+ */
+export function videoBidIsValid(bid) {
```

Comment:
Comment made above for function `mediaTypesValid` applies here too.
Also add missing JSDoc for param and return value and unit test for this function